spec/views/lists/_list.html.erb_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "lists/_list" do
+  context "when the list is empty" do
+    it "shows an appropriate message" do
+      list = instance_double(List, empty?: true, name: "")
+
+      render template: "lists/_list", locals: { list: list }
+
+      expect(rendered).to have_css("li", text: t(".list.empty"))
+    end
+  end
+
+  context "when there are books to list" do
+    it "lists the books" do
```

Comment:
This duplicates the partial's tests, any thoughts on a better way to approach this?